webpack/global_index.js

```diff
@@ -45,3 +46,6 @@ addGlobalFill(
 addGlobalFill('details-cards', 'Installable errata', <ErrataOverviewCard key="errata-overview" />, 1900);
 addGlobalFill('host-tab-details-cards', 'Installed products', <InstalledProductsCard key="installed-products" />, 100);
 addGlobalFill('host-tab-details-cards', 'Registration details', <RegistrationCard key="registration-details" />, 200);
+
+// Details tab cards & card extensions
+addGlobalFill('host-details-tab-properties-1', 'Subscription UUID', <SystemPropertiesCardExtensions key="subscription-uuid" />);
```

Comment:
Curious about the card name. Is there going to be `host-details-tab-properties-2` later?